docs/source/tutorial/basic.rst

```diff
@@ -112,9 +124,19 @@ In order to preserve gradient information, pass the ``retain_grad`` argument to
    >>> z.grad
    array([-1.], dtype=float32)
 
-All these computations are easily generalized to a multi-element array input.
+Otherwise, ``z.grad`` will be ``None`` as follows:
+
+.. doctest::
+
+   >>> y.backward()  # The default value of retain_grad is False
+   >>> z.grad is None
+   True
+
+All these computations are easily generalized to multi-element array input.
 Note that if we want to start backward computation from a variable holding a multi-element array, we must set the *initial error* manually.
-This is done simply by setting the :attr:`~Variable.grad` attribute of the output variable:
+Because when the :attr:`~Variable.size` of a variable (it means the number of elements in the array) is ``1``, it's considered as a variable object that represents a loss value, so that the :attr:`~Variable.grad` attribute of the variable is automatically filled with ``1``.
+On the other hand, when the size of a variable is larger than ``1``, the :attr:`~Variable.grad` attribute remains ``None``, so that we need to give *initial error* explicitly by hand before :meth:`~Variable.backward`.
```

Comment:
remains ``None``, and it is necessary to set the *initial error* explicitly before running :meth:`~Variable.backward`.